lib/note-list/index.jsx

```diff
@@ -413,43 +413,51 @@ export class NoteList extends Component {
       isSmallScreen,
     });
 
+    const isEmptyList = notes.length === 0;
+
+    const emptyTrashButton = (
```

Comment:
This change probably caused this issue: https://github.com/Automattic/simplenote-electron/pull/857#issuecomment-421426835